src/suit/js/SUIT.js

```diff
@@ -70,8 +73,10 @@ if (tapUrl) { // if a url is produced with confidence put it at the top otherwis
 }
 
 if (!tapUrl || !confident) {
-    setTimeout(
-        () => showInfoPopup('Could not auto-detect the location of the TAP service for this LSP instance.'), 5000)
+    setTimeout( () =>
+            showInfoPopup(
+                `Could not auto-detect the location of the TAP service for this LSP instance using url: ${window.location.href}`),
```

Comment:
I'd prefer:
`Could not infer the location of the TAP service for this LSP instance from the window URL: `